src/containers/Runs/Runs.js

```diff
@@ -196,7 +214,44 @@ function Runs({ intl }) {
     }
 
     return [
-      // TODO: rerun?
+      {
+        action: rerun,
+        actionText: intl.formatMessage({
+          id: 'dashboard.rerun.actionText',
+          defaultMessage: 'Rerun'
+        }),
+        disable: resource => !!resource.metadata.labels?.['tekton.dev/pipeline']
+      },
+      {
+        actionText: intl.formatMessage({
+          id: 'dashboard.cancelTaskRun.actionText',
+          defaultMessage: 'Stop'
+        }),
+        action: cancel,
+        disable: resource => {
+          const { status } = getStatus(resource);
+          return status && status !== 'Unknown';
```

Comment:
Should this option be disabled if `!isRunning(...)`? or is this check equivalent?

Reply:
`isRunning` is a little different. It only works for `TaskRun` and `PipelineRun` resources as we determine they're running by looking for the combination of `status === 'Unknown'` AND a known reason (e.g. `Running`, `Stopping`, etc.)

For `Run` resources, any `status === 'Unknown'` is considered running. If status is unset it means the controller hasn't started processing it yet, in which case we should also be allowed to prevent it from running.

I realise I mistakenly used `isRunning` for the delete action so I've updated to remove that and correctly check for `status === 'Unknown'` instead.